infra/build_deploy_config.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import os
+import boto3
+import argparse
+
+
+infra_root = os.path.abspath(os.path.dirname(__file__))
+
+
+parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
Please import each module using the full pathname location of the module:

`from argparse import ArgumentParser`

Reference to style guide: http://google.github.io/styleguide/pyguide.html#23-packages